flink/v1.17/flink/src/main/java/org/apache/iceberg/flink/source/IcebergSource.java

```diff
@@ -216,6 +225,8 @@ public static class Builder<T> {
     private Table table;
     private SplitAssignerFactory splitAssignerFactory;
     private SerializableComparator<IcebergSourceSplit> splitComparator;
+    private IcebergEventTimeExtractor timeExtractor;
```

Comment:
nit: I would rename `timeExtractor` -> `eventTimeExtractor` to be more inline with the class names.

Reply:
Will fix to conform the current code, but I try to avoid having long variable names where it doesn't add value.

Reply:
done